disnake/abc.py

```diff
@@ -1334,7 +1343,14 @@ async def send(
 
             .. versionadded:: 1.6
         view: :class:`disnake.ui.View`
-            A Discord UI View to add to the message.
+            A Discord UI View to add to the message. This can not be specified together with ``components``.
+
+            .. versionadded:: 2.0
+
+        components: Any
+            A list of components to include in the message. This can not be specified together with ``view``.
```

Comment:
If this parameter also accepts a single `Item`, `list of components` isn't entirely accurate (other cases below)

Reply:
How about using: "The components to include in the message, it can be a :class:`List` of components or a single one. This cannot ..."?